### PR TITLE
Settings UI

### PR DESCRIPTION
**Pull request by graphemecluster**
For your reference.

### CODE REVIEW

File: Cantoboard/Extension/URL+Extension.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//  Cantoboard
+//
+//  Created by Alex Man on 9/12/21.
+//
+
+import UIKit
+import AVFoundation
+
+extension URL {
+    var videoAspectRatio: CGFloat? {
```
Comment (alex-the-man) [inline Cantoboard/Extension/URL+Extension.swift:12]: This operation looks super expensive. Could we cache them instead?

**Comment by alex-the-man** [comment]
介唔介意拆呢個PR做細啲嘅PR？每一個PR一個purpose？

**Comment by graphemecluster** [comment]
For your reference only, actually. Many of them are trivial changes aiming to make the code style consistent.

File: Cantoboard/Extension/URL+Extension.swift (hunk 1 above)
Comment (graphemecluster) [reply Cantoboard/Extension/URL+Extension.swift:12]: Since all the video are in the same size currently, hard-code it until then.

**Pull request closed without merging**